Pod/Classes/Model/PhotoCollectionViewDataSource.swift

```diff
@@ -64,6 +64,7 @@ final class PhotoCollectionViewDataSource : NSObject, UICollectionViewDataSource
         UIView.setAnimationsEnabled(false)
         let cell = collectionView.dequeueReusableCell(withReuseIdentifier: photoCellIdentifier, for: indexPath) as! PhotoCell
         cell.accessibilityIdentifier = "photo_cell_\(indexPath.item)"
+        cell.isAccessibilityElement = true
```

Comment:
Curious as to why this needs to be specified when the docs say it's the default for UIKit controls?
https://developer.apple.com/documentation/objectivec/nsobject/1615141-isaccessibilityelement

Either way, sweet fix! 😄

Reply:
Oh, is it because a cell isn't a control?

Reply:
That’s right, cells don’t subclass UIControl.

Reply:
I found it through some SO-diving.